src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -20,7 +24,17 @@ public void actionPerformed(ActionEvent e) {
 			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
 		}else if(e.getActionCommand().equals("Figure Joe 2")) {
 			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+		}else if(e.getActionCommand().equals("Square")) {
+			FactoryInterface factory = new SquareFactory();
+			factory.create().execute(driverManager.getCurrentDriver());
+		}else if(e.getActionCommand().equals("Triangle")) {
+			FactoryInterface factory = new TriangleFactory();
+			factory.create().execute(driverManager.getCurrentDriver());
+		}else if(e.getActionCommand().equals("Rectangle")) {
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.